Loop/AppDelegate.swift

```diff
@@ -27,62 +27,108 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, DeviceOrientationCo
     private var bluetoothStateManager: BluetoothStateManager!
 
     var window: UIWindow?
+    
+    var launchOptions: [UIApplication.LaunchOptionsKey: Any]?
 
     private var rootViewController: RootNavigationController! {
         return window?.rootViewController as? RootNavigationController
     }
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    
+    private var isAfterFirstUnlock: Bool {
+        let fileManager = FileManager.default
+        do {
+            let documentDirectory = try fileManager.url(for: .documentDirectory, in: .userDomainMask, appropriateFor:nil, create:false)
+            let fileURL = documentDirectory.appendingPathComponent("protection.test")
+            guard fileManager.fileExists(atPath: fileURL.path) else {
+                let contents = Data("unimportant".utf8)
```

Comment:
[nit] Write empty data?

Reply:
Not sure what you're asking here.

Reply:
Sorry, rather than writing the string "unimportant" just write `Data()`. No big deal, though.